helpers/contexts/session.ts

```diff
@@ -2,8 +2,7 @@ import { createContext } from 'react'
 import { SessionData } from '../useSession'
 
 const SessionContext = createContext<SessionData>({
-  data: null,
-  error: null
+  data: { success: false }
```

Comment:
this may make sense for you now, but I think in the future it may confuse other devs because you are never setting it to `success: true`

Reply:
yeah it would be ideal to have it match what is returned from the `/session` endpoint in the future :)